### PR TITLE
Restore some features that appear in files and file lists

### LINKED ISSUE #3945: Copy button doesn't show (and other file-related features)
E.g.: https://github.com/TRSasasusu/SpikingMoYF/blob/master/network.py
image

### PR DESCRIPTION
**Pull request by fregante**
Fixes #3945 

Missing:

- [x] `forked-to`
- [x] `fork-source-link-same-view`
- [ ] any more features affected? Probably every isRepoTree/isSingleFile

## Test 1

gif

1. Visit https://github.com/sindresorhus/eslint-plugin-unicorn/tree/main/docs
2. Notice `edit-files-faster` and `latest-tag-button`
3. Enter the folder
4. The same features are still working
5. Open a markdown file
6. Notice `view-markdown-source` (wrong position due to https://github.com/sindresorhus/refined-github/pull/4243)

## Test 2

gif

1. Visit https://github.com/sindresorhus/refined-github/tree/main/source/features
2. Open a tsx file
3. Notice `copy-file` and `rgh-feature-description`

## Test 3

gif

1. Visit https://github.com/sindresorhus/refined-github/tree/main/source
2. Enter a folder
3. Notice `download-folder-button` and `default-branch-button`

### CODE REVIEW

**Comment by yakov116** [comment]
So far I found
`same-page-definition-jump`
`html-preview-link`
`list-prs-for-file`

**Comment by fregante** [comment]
It's starting to feel like it's a lot 😂 I'm glad I'm adding the selector checker so I don't have to clutter every `init` with `if !exists(), return`

**Comment by yakov116** [comment]
> * [ ] [`forked-to`](/sindresorhus/refined-github/blob/main/source/features/forked-to.tsx)
> * [ ]  [`fork-source-link-same-view`](/sindresorhus/refined-github/blob/main/source/features/fork-source-link-same-view.tsx)

@fregante whats the plan for these 2?

File: source/features/html-preview-link.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ function init(): void {
 		.parentElement! // `BtnGroup`
 		.prepend(
 			<a
-				className="btn btn-sm BtnGroup-item"
+				className="btn btn-sm BtnGroup-item html-preview-link"
```
Comment (fregante) [inline source/features/html-preview-link.tsx]: Don't forget .rgh

**Review by fregante — COMMENTED** [review]
> `same-page-definition-jump`

How is that affected? It uses `delegate`, it never had to be called multiple times

**Comment by fregante** [comment]
As mentioned, all `isRepoTree`/`isSingleFile` are affected, but I'll merge this for now to create a release today.

**Comment by fregante** [comment]
As mentioned, all `isRepoTree`/`isSingleFile` are affected, but I'll merge this for now to create a release today.

**Comment by yakov116** [comment]
> > `same-page-definition-jump`
> 
> How is that affected? It uses `delegate`, it never had to be called multiple times

Its not called multiple times. It never gets activated in the first place. The same way all the other features didn't load unless you refreshed the page.

**Pull request merged**